### DISCUSSION TITLE
Please release source code for binary blob of Lib

### DISCUSSION THREAD

**Discussion by iamtimmy** [NewWorldComingSoon/llvm-msvc-issues#23] [Q&A]
[lld/COFF/llvmintrinsicrewrite_x64_lib.h](https://github.com/NewWorldComingSoon/llvm-msvc/blob/a00c32f96787268fd70f2ba17c33188c5275ece1/lld/COFF/llvmintrinsicrewrite_x64_lib.h)

Why is this in here like this instead of as source?

**Comment by gmh5225** [comment]
Sorry to see it so late, because this is a forced linked library file.
You can generate llvmintrinsicrewrite_x64/X32_lib.h by converting compiled library files into header files via bin2c.

source:
https://github.com/gmh5225/LLVMIntrinsicRewrite
